tests/ops/functions/test_generator.py

```diff
@@ -402,14 +402,14 @@ def test_observable_no_coeff(self):
         assert isinstance(gen, qml.Hamiltonian)
         assert gen.compare(qml.Hamiltonian([1.0], [qml.PhaseShift(0.5, wires=0).generator()]))
 
-    @pytest.mark.usefixtures("use_legacy_opmath")
+    @pytest.mark.usefixtures("legacy_opmath_only")

```

Comment:
Why is this only tested with legacy opmath?

Reply:
Fixed.